security/pkg/nodeagent/cache/secretcache.go

```diff
@@ -463,13 +465,17 @@ func (sc *SecretManagerClient) keyCertSecretItem(cert, key, resource string) (*s
 // readFileWithTimeout reads the given file with timeout. It returns error
 // if it is not able to read file after timeout.
 func (sc *SecretManagerClient) readFileWithTimeout(path string) ([]byte, error) {
-	retryBackoffInMS := int64(firstRetryBackOffInMilliSec)
+	retryBackoffInMS := int64(firstRetryBackOffInMilliSec * time.Millisecond)
 	timeout := time.After(totalTimeout)
 	for {
 		cert, err := os.ReadFile(path)
 		if err == nil {
 			return cert, nil
 		}
+		// permanent error
```

Comment:
isn't this the expected error? we are retrying because it may not exist yet?

Reply:
This is inspired by `generateKeyCertFromExistingFiles`, we checked not exist error there too